src/components/NavbarDrawer.js

```diff
@@ -134,7 +216,7 @@ function NavbarDrawer({ updateTT, toggleTheme }) {
               keepMounted: true, // Better open performance on mobile.
             }}
           >
-            {drawer}
+        {drawerlist}
```

Comment:
Indentation issues like this need to be fixed, ESLint should point all of them out to you